vdr/validators.go

```diff
@@ -24,10 +41,15 @@ type serviceValidator struct{}
 
 func (s serviceValidator) Validate(document did.Document) error {
 	knownServiceIDs := make(map[string]bool, 0)
+	knownServiceTypes := make(map[string]bool, 0)
 	for _, method := range document.Service {
 		if err := verifyDocumentEntryID(document.ID, method.ID, knownServiceIDs); err != nil {
 			return fmt.Errorf("invalid service: %w", err)
 		}
+		if knownServiceTypes[method.Type] {
+			return errors.New("invalid service: duplicate service type")
```

Comment:
make it a `ErrDuplicateService` so you can return a 400 when called from API?

Reply:
also relates to other PR

Reply:
Will fix this in https://github.com/nuts-foundation/nuts-node/pull/313 since API call if only possible through that PR